main_test.go

```diff
@@ -58,11 +79,12 @@ func TestReLoginFailed(t *testing.T) {
 	_, status, err := wrongClient.Request("/status/code/1")
 	g.Expect(err).NotTo(BeNil())
 	g.Expect(status.ResponseType).To(Equal("Error"))
-	g.Expect(status.Response).To(Equal("re-login failed"))
+	// This test returns one of two different values based on the  API version, either: 'request failed' or 'Invalid sessionkey'
+	g.Expect(status.Response).Should(BeElementOf([]string{"request failed", "Invalid sessionkey"}))
```

Comment:
I don't get the same result on my appliance. I actually get `re-login failed`. Could it be because of a different version of the api running on the appliance? The `re-login failed` message actually comes from the code of the api client itself, it occurs when a session key expire and that the login fail.

Reply:
If you let me know the version of software running on your storage controller I can verify what the API service is returning. For now, I also added 're-login failed' as one of three possible return strings for the various revisions of the API return values. The 'show versions detail' CLI command should return what I need to look up the firmware. Thank you.

Reply:
Looks good. Here is the output I get from the `show versions detail` command:
```json
{
    "versions": [
        {
            "object-name": "controller-a-versions",
            "meta": "/meta/versions",
            "sc-cpu-type": "Broadwell 2200MHz",
            "bundle-version": "VL270P004",
            "bundle-base-version": "V270",
            "build-date": "Tue Jul 21 10:33:07 MDT 2020",
            "sc-fw": "VLS270R01-07",
            "sc-baselevel": "VLS270R01-07",
            "sc-memory": "N/A",
            "sc-fu-version": "6.30.27744x",
            "sc-loader": "31.000",
            "capi-version": "3.20",
            "mc-fw": "VXM270P003-11",
            "mc-loader": "6.30.27744x",
            "mc-base-fw": "VXM270P003-11",
            "fw-default-platform-brand": "HPE",
            "fw-default-platform-brand-numeric": 15,
            "ec-fw": "3207",
            "pld-rev": "1.9",
            "prm-version": "N/A",
            "hw-rev": "3.1",
            "him-rev": "11",
            "him-model": "6",
            "backplane-type": 7,
            "host-channel_revision": 2,
            "disk-channel_revision": 3,
            "mrc-version": "0.3.2.11",
            "ctk-version": "No CTK present"
        },
        {
            "object-name": "controller-b-versions",
            "meta": "/meta/versions",
            "sc-cpu-type": "Broadwell 2200MHz",
            "bundle-version": "VL270P004",
            "bundle-base-version": "V270",
            "build-date": "Tue Jul 21 10:33:07 MDT 2020",
            "sc-fw": "VLS270R01-07",
            "sc-baselevel": "VLS270R01-07",
            "sc-memory": "N/A",
            "sc-fu-version": "6.30.27744x",
            "sc-loader": "31.000",
            "capi-version": "3.20",
            "mc-fw": "VXM270P003-11",
            "mc-loader": "6.30.27744x",
            "mc-base-fw": "VXM270P003-11",
            "fw-default-platform-brand": "HPE",
            "fw-default-platform-brand-numeric": 15,
            "ec-fw": "3207",
            "pld-rev": "1.9",
            "prm-version": "N/A",
            "hw-rev": "3.1",
            "him-rev": "11",
            "him-model": "6",
            "backplane-type": 7,
            "host-channel_revision": 2,
            "disk-channel_revision": 3,
            "mrc-version": "0.3.2.11",
            "ctk-version": "No CTK present"
        }
    ],
    "status": [
        {
            "object-name": "status",
            "meta": "/meta/status",
            "response-type": "Success",
            "response-type-numeric": 0,
            "response": "Command completed successfully. (2021-06-02 15:46:53)",
            "return-code": 0,
            "component-id": "",
            "time-stamp": "2021-06-02 15:46:53",
            "time-stamp-numeric": 1622648813
        }
    ]
}
```